dev/Dockerfile.prestosql.340

```diff
@@ -0,0 +1,25 @@
+FROM openjdk:11-jdk-slim-buster
```

Comment:
Borrowed from [this buddy's Github](https://github.com/vimagick/dockerfiles/tree/master/prestosql)